File: 2.0/java/me/getApiCredential.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.account.model.ApiCredentials;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("account_login", "account_password");
        // you can use 'fields' argument to limit returned fields
        ApiCredentials credentials = callfireClient.meApi().getApiCredentials(123456L, "name,enabled");
    }
}

File: 2.0/csharp/me/getApiCredential.cs
using CallfireApiClient;
using CallfireApiClient.Api.Account.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("account_login", "account_password");
        // add 2nd argument to limit returned fields
        ApiCredentials credentials = client.MeApi.GetApiCredentials(379506003, "name,enabled");
    }
}
